### PR TITLE
Fix usbserial function names

### PR DESCRIPTION
**Pull request by geekbozu**
Requested per Davee

### CODE REVIEW

**Comment by geekbozu** [comment]
i guess how this was merged last time made everything show back up again...idk

**Comment by d3m3vilurr** [comment]
rebase last commit :(
actually this pr not need all things, just need last commit.
```
git remote add upstream https://github.com/vitasdk/vita-headers
git fetch upstream
git checkout master
git reset --hard upstream/master
git cherry-pick a719900
git push origin master -f
```

File: db.json [hunk 1]
```diff
@@ -5533,12 +5533,12 @@
 		"variables": {},
 		"functions":{
 			"sceUsbSerialStartForDriver": 4276614330,
```
Comment (d3m3vilurr) [inline db.json]: it's remain ForDriver. intentional changes?

File: include/psp2kern/usbserial.h [hunk 2]
```diff
@@ -33,26 +33,26 @@ int sceUsbSerialStartForDriver(void);
...
-int sceUsbSerialCloseForDriver(void);
+int sceUsbSerialCloseForKernel(void);
 
 /**
  * Get receive buffer length
  *
  * @return Returns length of USB rx buffer
  */
-unsigned int sceUsbSerialGetLenForDriver(void);
+unsigned int sceUsbSerialGetLenForKernel(void);
```
Comment (DaveeFTW) [inline include/psp2kern/usbserial.h]: rename to ```sceUsbSerialGetReceiveBufferSizeForKernel```? or maybe ``sceUsbSerialGetRecvBufferSizeForKernel```? Names getting long with the ```ForKernel```  prefix. @d3m3vilurr any ideas on a better name?

Comment (geekbozu) [inline include/psp2kern/usbserial.h]: Oh yeah we wanted to change that. 
sceUsbSerialGetRXLenForKernel is what comes to mind. To me

Comment (geekbozu) [inline include/psp2kern/usbserial.h]: pending me being the human error causing machine. Should be good.

**Pull request merged**